meta-balena-common/recipes-containers/mkfs-hostapp-native/files/Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM debian:stretch
+FROM debian:bullseye
 
 VOLUME /mnt/sysroot/inactive
 
 RUN apt-get update && apt-get install -y \
 	ca-certificates \
-	iptables
+	iptables \
+	btrfs-progs \
+	duperemove \
+	rsync
 
-COPY create /usr/bin/
+# Fixes issue with Docker expecting iptables when the distro is using nftables
+RUN update-alternatives --set iptables /usr/sbin/iptables-legacy
 
-CMD [ "/usr/bin/create" ]
```

Comment:
Maybe keep the default CMD in case it's part of another workflow and just change it to `"/usr/bin/create.ext4"`?

Reply:
Fixed